objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/ObjRecTestBase.java

```diff
@@ -25,12 +25,24 @@
     public static final String JPEG_IMAGE_LOCATION = System.getProperty("buildDir") + "/testResources/sampleImage.jpg";
     public static final String PNG_IMAGE_LOCATION = System.getProperty("buildDir") + "/testResources/sampleImage.png";
     public static final String VIDEO_LOCATION = System.getProperty("buildDir") + "/testResources/sampleVideo-1.0.mov";
+    // Used to test suppressNullValues. Finding online cameras with nothing on them is hard, so we'll
+    // simply include a video of a white wall.  This should be good enough to find nothing.
```

Comment:
Common subclass of tests in which to define the camera URL (now only one since we use a file-based video for the video of nothing).